backend/app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+    public function index()
+    {
+        return Category::all();
+    }
+
+    // Create a Category
+    public function store(Request $request)
+    {
+        $request->validate([
+            'title' => ['required', 'unique:categories', 'max:225'],
+            'description' => ['required'],
+        ]);
+        Category::create([
+            'title' => $request['title'],
+            'description' => $request['description'],
+        ]);
+
+        return response()->json(['message' => 'Category Created!'], 201);
+    }
+
+    // Display Description
+    public function show(Request $request)
+    {
+        $request->validate([
+            'title' => ['required'],
+        ]);
+
+        return Category::where('title', $request->title)->first()->description;
```

Comment:
Do this to all model
```suggestion
        $request->validate([
            'title' => ['required'],
        ]);
        
        $category = Category::where('title', $request->title)->first();

        return response()->json([
          'category' => $category,
        ]);
```